pennylane_lightning_kokkos/src/algorithms/ObservablesKokkos.hpp

```diff
@@ -0,0 +1,520 @@
+#pragma once
+
+#include <functional>
```

Comment:
Are you directly using the `functional` library here?
If not, I suggest removing this line.
```suggestion
```

Reply:
yea, let me remove it.